app/internal/place/client.go

```diff
@@ -78,6 +89,10 @@ func (c *Client) LookupPostcode(ctx context.Context, postcode string) ([]Address
 		return []Address{}, err
 	}
 
+	if postcodeLookupResponse.Error.Statuscode == invalidPostcodeCode {
```

Comment:
`invalidPostcodeCode` is just `http.StatusBadRequest`, so also should the error be `BadRequestError` since it isn't necessarily an invalid postcode?

Reply:
That's true - it's a shame the API doesn't allow us to differentiate any further than this. I'll update the error type name so its more generic.

Reply:
yep, I'd love them to return a better body than just "message"